src/Keboola/Console/Command/ProjectsAddFeature.php

```diff
@@ -12,57 +10,139 @@
-                    $manageClient->addProjectFeature($projectId, $feature);
-                    $output->writeln("Feature '{$feature}' successfully added.");
+                    $output->writeln(sprintf(' - feature "%s" DOES NOT exist in the project %s yet. Enable force mode with -f option', $featureName, $projectInfo['id']));
+                }
+                $this->projectsUpdated++;
+            }
+        }
+        $output->write("\n");
+    }
+
+    protected function addFeatureToAllProjects(Client $client, OutputInterface $output, string $feature, bool $force): void
+    {
+        $maintainers = $client->listMaintainers();
+        foreach ($maintainers as $maintainer) {
+            $this->maintainersChecked++;
+            $organizations = $client->listMaintainerOrganizations($maintainer['id']);
+            foreach ($organizations as $organization) {
+                $this->orgsChecked++;
+
+                $projects = $client->listOrganizationProjects($organization['id']);
+                foreach ($projects as $project) {
+                    $this->addFeatureToProject($client, $output, $project, $feature, $force);
                 }
             }
-            $output->write("\n");
         }
     }
+
+    protected function addFeatureToSelectedProjects(Client $client, OutputInterface $output, string $featureName, array $projectIds, bool $force): void
+    {
+        foreach ($projectIds as $projectId) {
+            $project = $client->getProject($projectId);
+            $this->addFeatureToProject($client, $output, $project, $featureName, $force);
+        }
+    }
+
+    public function checkIfFeatureExists(Client $client, string $featureName): bool
+    {
+        $projects = $client->listFeatures(['type' => 'project']);
+        foreach ($projects as $project) {
+            if ($project['name'] === $featureName) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    public function execute(InputInterface $input, OutputInterface $output): ?int
+    {
+        $args = $input->getArguments();
+        $force = $input->getOption(self::OPT_FORCE);
+        $featureName = $args[self::ARG_FEATURE];
+        $projectsOption = $args[self::ARG_PROJECTS];
+
+        $client = $this->createClient($args[self::ARG_URL], $args[self::ARG_TOKEN]);
+
+
+        if (!$this->checkIfFeatureExists($client, $featureName)) {
+            $output->writeln(sprintf('Feature %s does NOT exist', $featureName));
+            return 1;
+        }
+        if (strtolower($projectsOption) === 'all') {
+            $this->addFeatureToAllProjects($client, $output, $featureName, $force);
+        } else {
+            $projectIds = array_filter(explode(',', $projectsOption), 'is_numeric');
+            $this->addFeatureToSelectedProjects($client, $output, $featureName, $projectIds, $force);
+        }
+
+        $output->writeln(
+            sprintf(
+                "Checked %d maintainers \n"
```

Comment:
Trochu ma zaskocilo preco je checkovani maintainer a org potom som si uvedomil, ze je to preto pretoze som tam dal presne idcka projektov tazke to neforeachuje vsetko. Mozno by som este popremyslal nad tim ze by sa tie informacie vypisovali len ak je parameter `all`